packages/nimble-components/build/generate-workers/source/render-worker.ts

```diff
@@ -0,0 +1,44 @@
+import * as Comlink from 'comlink';
+import { HealthStatus, healthStatus } from './health-status';
+
+export class RenderWorker {
+    private worker!: number;
+    private dieMatrix: Uint8Array = Uint8Array.from([]);
+    private performanceTest: string | undefined;
+    private healthStatus: HealthStatus = healthStatus.unknown;
+
+    constructor() {
+    }
+
+    public emptyMatrix(): void {
+        this.dieMatrix = Uint8Array.from([]);;
+    }
+
+    public updateMatrix(
+        data: Iterable<number>
+    ): void {
+        const start = this.performanceTest !== undefined ? self.performance.now() : undefined;
+        this.dieMatrix = Uint8Array.from(data);
+        if (this.performanceTest !== undefined) {
+            self.performance.measure(`${this.performanceTest} - worker:${this.worker} - renderDies`, { start });
+        }
+    }
+
+    public areMethodsCallable(): boolean {
```

Comment:
I don't understand the purpose of this method. When would the tested condition ever be false?

Reply:
The code from render-worker will be built, packed and later moved to a different location to be used. We wanted a way to check that the render code is reachable. (maybe it doesn't make sense indeed)
![image](https://github.com/ni/nimble/assets/110180309/60807502-c583-4a36-9567-de5687385bff)
this is how the code looks in the end

Reply:
I decided to delete all the healthcheck related code and rely on unit tests to ensure the code still works after bundling with rollup, now that I think of this it seems indeed a not so good idea